Sources/Plasma/FeatureLib/pfPython/pyGameCliGlue.cpp

```diff
@@ -40,11 +40,11 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 *==LICENSE==*/
 
+#include <Python.h>
 #include "pyGameCli.h"
 
 #include "plPythonConvert.h"
 
-#include <Python.h>
```

Comment:
Hmm, pyGlueHelpers.h looks like it was meant to work without Python.h included, based on the manual definitions of `PyObject`, etc., but the `#if PY_VERSION_HEX` breaks this principle.

I assume the goal was to avoid indirectly including Python.h in all of the pfPython header files. I guess the proper fix would be to split pyGlueHelpers.h into two headers (one that works without Python.h, for use by other headers, and one that includes Python.h, for use by the implementation code), but that would be a larger change obviously.